keras_cv/models/clip/clip_image_encoder.py

```diff
@@ -0,0 +1,58 @@
+# encode images
+from collections import OrderedDict
+from typing import Tuple
+from typing import Union
+
+import numpy as np
+import tensorflow as tf
+import torch
+import torch.nn.functional as F
+from tensorflow.keras import layers
+from torch import nn
+
+from deepvision.layers.clip_patching_and_embedding import CLIPPatchingAndEmbedding
```

Comment:
I'm a little confused. Why is this a TF only version? Why are we importing Torch modules?

Reply:
Comment in the main thread 🙇